aiida/orm/implementation/sqlalchemy/computers.py

```diff
@@ -60,7 +60,7 @@ def copy(self):
         make_transient(dbcomputer)
         session.add(dbcomputer)
 
-        newobject = self.__class__.from_dbmodel(dbcomputer)  # pylint: disable=no-value-for-parameter
+        newobject = self.__class__.from_dbmodel(dbcomputer, self.backend)  # pylint: disable=no-value-for-parameter
```

Comment:
Think the pylint comment can be removed now that you pass the backend?
```suggestion
        newobject = self.__class__.from_dbmodel(dbcomputer, self.backend)
```